packages/sdk/src/StreamrClient.ts

```diff
@@ -462,14 +466,14 @@ export class StreamrClient {
      * Grants permissions on a given stream.
      */
     grantPermissions(streamIdOrPath: string, ...assignments: PermissionAssignment[]): Promise<void> {
-        return this.streamRegistry.grantPermissions(streamIdOrPath, ...assignments)
+        return this.streamRegistry.grantPermissions(streamIdOrPath, ...assignments.map((a) => toInternalPermissionAssignment(a)))
```

Comment:
Stylistic choice, you can just pass function directly
```suggestion
        return this.streamRegistry.grantPermissions(streamIdOrPath, ...assignments.map(toInternalPermissionAssignment))
```

Reply:
Fixed